### PR TITLE
Error page

### PR DESCRIPTION
**Pull request by angiecesere**
@anjanette Here's my changes! do you want to code review them really quick to make sure I didn't do anything crazy?

### CODE REVIEW

File: styles/_base.scss [hunk 1]
```diff
@@ -170,6 +178,16 @@ ul {
   @include max-width-container;
 }
 
+.fit {
+  max-width: 500px;
```
Comment (anjanette) [inline styles/_base.scss]: I think we should resize the image and drop this class. I should probably do that on the live Artstor site as well.

File: styles/_base.scss [hunk 2]
```diff
@@ -105,6 +105,10 @@ select {
   }
 }
 
+.page-title {
+  font-size: 46px;
```
[the remark this answers is not in the document]
Comment (anjanette) [reply styles/_base.scss]: If this is a class only for the error page, then we might want to rethink this

File: templates/error_page.hbs [hunk 3]
```diff
@@ -10,12 +9,41 @@
...
+{{#is error 'not_found'}}
+  <h1 class="page-title compact-bottom">404</h1>
+  <h2 class="compact-top">Page not found</h2>
+  <p>The link you followed may be broken or we may have moved the page. If you cannot find what you are looking for, <a href="https://www.artstor.org/contact-us">contact us</a>.
+  <p>
+  	<a href="/" class="button">Back to Homepage</a>
+  	<a href="http://support.artstor.org/" class="button hollow" target="_blank">Get Help</a>
+  </p>
+<div>
+  <img class="fit" src="https://theme.zdassets.com/theme_assets/1006386/1afd840bddc97c7702e5d701be71eba177e0a776.jpg" alt="Philippe Mercier, The Sense of Sight, 1744 – 1747. Yale Center for British Art">
```
Comment (anjanette) [inline templates/error_page.hbs]: let's resize the image and remove the fit class

File: templates/error_page.hbs [hunk 4]
```diff
@@ -1,6 +1,5 @@
 <div class="container-divider"></div>
-<div class="error-page">
-  <h1>{{t 'oops'}}</h1>
+<div class="error-page text-center">
```
Comment (anjanette) [inline templates/error_page.hbs]: I think the error page needs more spacing at the top of the page

maybe we can do 
.error-page {margin-top: 60px;}

File: templates/error_page.hbs [hunk 5]
```diff
@@ -10,12 +9,41 @@
...
-
-  {{#link 'help_center'}}
-    {{t 'back_to_homepage'}}
-  {{/link}}
+{{#is error 'not_found'}}
+  <h1 class="page-title compact-bottom">404</h1>
+  <h2 class="compact-top">Page not found</h2>
+  <p>The link you followed may be broken or we may have moved the page. If you cannot find what you are looking for, <a href="https://www.artstor.org/contact-us">contact us</a>.
+  <p>
+  	<a href="/" class="button">Back to Homepage</a>
```
Comment (anjanette) [inline templates/error_page.hbs]: Let's think about what buttons are useful here - right now, these both go to the same page. Do we just need one button? or do we need one that goes to the support home and one that goes to the platform home? May be a good question for Bonnie and Kelly

File: styles/_base.scss (hunk 1 above)
Comment (angiecesere) [reply styles/_base.scss]: @anjanette yeah that makes a lot of sense.

File: templates/error_page.hbs (hunk 3 above)
Comment (angiecesere) [reply templates/error_page.hbs]: @anjanette got it

**Pull request merged**